share/src/main/java/io/zensoft/share/service/facebook/FacebookPageAccessTokenRetriever.java

```diff
@@ -11,19 +12,22 @@
 import java.util.Map;
 
 @Service
+@Slf4j
 public class FacebookPageAccessTokenRetriever {
     private String userAccessToken;
     private String pageAccessToken;
 
     public String getZensoftPageAccessToken (){
```

Comment:
does this method need to log about successful execution?
'cause it logs when it starts `log.info("get Zensoft's page access token by pre-assigned user access token");`, and if catches an error, but there's nothing about successful execution

Reply:
If this log is needed, does it mean, that we should add logs to both begin and end of all methods?

Reply:
I think, it makes sense to add logs at the begining and at the end of each method.